src/UserRepository/Htpasswd.php

```diff
@@ -61,10 +65,12 @@ public function authenticate(string $credential, string $password = null) : ?Use
         }
         fclose($handle);
 
-        return $found
-            && password_verify($password === null ? '' : $password, $hash)
-                ? $this->generateUser($credential)
-                : null;
+        if ($found && password_verify($password === null ? '' : $password, $hash)) {
```

Comment:
We can have here `$password ?? ''` instead of `$password === null ? '' : $password`.